### PR TITLE
Temporary update privacy and term of use page for tri-agency meeting

### PR DESCRIPTION
**Pull request by pengyin-shan**
Update privacy and term of use page temporary with English-only draft content for tri-agency meeting. Will update French content once the draft is finished.

### CODE REVIEW

File: app/views/layouts/_footer.html.erb [hunk 1]
```diff
@@ -68,8 +68,8 @@
         }, Rails.configuration.branding[:organisation][:url])%></li>
       <li>•</li>
       <li><a href="<%= contact_us %>"><%= _('Contact us') %></a></li>
-      <li><a href="<%= terms_of_use_path %>"><%= _('Terms of use') %></a></li>
-      <li><a href="<%= privacy_path %>"><%= _('Privacy statement') %></a></li>
+      <li><a href="/terms"><%= _('Terms of use') %></a></li>
+      <li><a href="/privacy"><%= _('Privacy statement') %></a></li>
```
Comment (lagoan) [inline app/views/layouts/_footer.html.erb:72]: We should not be hardcoding the URLs whenever possible. Is there a reason for this change?

File: app/views/static_pages/privacy.html.erb [hunk 2]
```diff
@@ -7,28 +7,88 @@
 <div class="row">
   <div class="col-md-12">
     <div>
-      <h2 class="fontsize-h3"><%= _("Information about you: how we use it and with whom we share it") %></h2>
-      <p><%= _("The information you provide will be used by the %{org_name} to offer you access to and personalisation of the %{application_name} service.") %
-             { application_name: ApplicationService.application_name, :org_name => Rails.configuration.x.organisation.names } %></p>
-      <p><%= _("The %{org_name} processes the personal data of %{application_name} users in order to deliver and improve the %{application_name} service in a customised manner and to ensure each user receives relevant information.") %
-             { application_name: ApplicationService.application_name, :org_name => Rails.configuration.x.organisation.name  } %></p>
-      <p><%= _("The processing of your personal data by the %{org_name} is necessary for pursuing the following legitimate interests:") %
-             { :application_name => ApplicationService.application_name, :org_name => Rails.configuration.x.organisation.name  } %></p>
```
Comment (lagoan) [inline app/views/static_pages/privacy.html.erb:16]: We should continue to use the `_()` method to handle the content translation, or are we handling this one differently?

File: app/views/layouts/_footer.html.erb (hunk 1 above)
Comment (pengyin-shan) [reply app/views/layouts/_footer.html.erb:72]: yeah, it used to link to an external portagenetwork site. now it needs to link to static page

File: app/views/static_pages/privacy.html.erb (hunk 2 above)
Comment (pengyin-shan) [reply app/views/static_pages/privacy.html.erb:16]: we don't have a french translation ready before the tri-agency meeting tomorrow. Thus Weiwei asked Alliance to use English only version for now, we will change back to the version with translation after the draft is finished

Comment (lagoan) [reply app/views/static_pages/privacy.html.erb:16]: So if I understand correctly, the text that is currently on this PR is still in the draft stage?

Comment (pengyin-shan) [reply app/views/static_pages/privacy.html.erb:16]: Yes, we need to get the draft term of use  & privacy release before tomorrow to prepare for tri-agency meeting. That;s why you see the code here is more on 'quick fix' style. Then the privacy office (I assume) will continue to finish the term of use and privacy, and give the French translation. Then I'll change the code back to `_()` method and release again. But for now, in order to prepare for tomorrow, we will release a 'drafted' English-only version.

**Review by portagenetwork — APPROVED** [review]
merge and release for now since the updated content needs to be ready before May 13th. Once the new privacy & term is ready, they will be published on the Github page and the link will be reversed.

**Pull request merged**